bin/sl-deploy.js

```diff
@@ -144,11 +150,20 @@ if (!local) {
 function exit(err, service) {
   var svc = service ? (service.name || service.id) : serviceName;
   if (err) {
-    console.error('Failed to deploy `%s` as `%s` to `%s` via `%s`',
-                  branchOrPack, svc, baseURL, exit.url);
```

Comment:
Why not? Its way more concise, and its concision makes it less error prone to use, particularly when converting from code that already uses positional args. Note the error you made at https://github.com/strongloop/strong-deploy/pull/54#discussion_r48898318. Its easy to do that kind of thing, and its particularly bad because there are ***three*** identical strings that must be identical, or the code is broken, and made worse because unit tests to cover this kind of thing are hard to build. The above is an error case, and we often don't test the exact message output from CLIs, particularly for errors, the tests are hard to build and fragile, but the effect is bugs like the above are easy to miss, and easy to create during what is a rote, bulk translation exercise.